lang-js/nvm/autobuild/build

```diff
@@ -1,13 +1,10 @@
-abinfo "Creating nvm profile..."
+abinfo "Installing nvm.sh, bash completion and doc..."
+install -Dvm644 nvm.sh "${PKGDIR}/usr/share/nvm/nvm.sh"
+install -Dvm644 bash_completion "${PKGDIR}/usr/share/bash-completion/completions/nvm"
+mkdir -pv "${PKGDIR}/usr/share/doc/nvm/"
+cp -v *.md "${PKGDIR}/usr/share/doc/nvm/"
```

Comment:
... Until here.

Reply:
Also, you could have just used `install -Dvm644 "$SRCDIR"/*.md -t "$PKGDIR"/usr/share/doc/nvm/`.